spec/factories/invoices.rb

```diff
@@ -12,6 +12,10 @@
 
     organization_sequential_id { rand(1_000_000) }
 
+    after :build do |invoice, values|
+      invoice.billing_entity = values.organization.default_billing_entity if invoice.billing_entity.nil?
```

Comment:
let's write idiomatic code 🙌 

```suggestion
      invoice.billing_entity = values.organization.default_billing_entity unless invoice.billing_entity
```